IP-POLICY-GUIDANCE.md

```diff
@@ -0,0 +1,190 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+
+# OpenJS Foundation IP Policy Guidance
+
+The purpose of this document is to provide an introduction to various aspects
+of the [OpenJS Foundation IP Policy](https://ip-policy.openjsf.org). Please
+note, if something appears inconsistent between the two documents, the IP
+Policy is always the canonical source of information. This document is merely
+meant to be interpretive, and changes to this document have no effect upon the
+[IP Policy](https://ip-policy.openjsf.org) unless the Board of Directors votes
+to accept them.
+
+## Essential Guidance
+
+We encourage everyone to [read the IP Policy](https://ip-policy.openjsf.org).
+It is intentionally short and to the point.
+
+At a high level, you will find the following key themes:
+
+1. _DO NOT_ change copyright lines unless you put them there _AND_ have the right
+to update them.  If you feel a need to add a _new_ copyright line, see the
+[copyright guidance](#copyrights-and-openjs-foundation-projects).
+1. You can use any of the open source licenses that are pre-approved in the
+[IP Policy](https://ip-policy.openjsf.org) without any additional approvals.
+1. If you need to use a license which isn't in the list, please ask the board.
+1. Your project website needs to have a
+[standard website footer](#copyright-notices-for-website-footers).
+1. If in doubt about anything related to IP, first read the
+[IP Policy](https://ip-policy.openjsf.org), and then
+[ask for help](#getting-help).
+
+There are also some optional things that projects can do which, while not
+mandatory, are referenced in the IP Policy or in this guidance. For example:
+
+1. You may update copyright lines that say "JS Foundation" or
+"Node.js Foundation" to "OpenJS Foundation".  This is a factual update
+reflecting the merged status of the organizations.
+1. You may decide to use the [CLA](https://cla.openjsf.org), or decide to stop
+using one.  This is optional under the OpenJS Foundation IP Policy, and is a
+decision that is to be made by the project's maintainers.
+1. Inserting [SPDX IDs](https://spdx.org/ids-how): While projects decide the
```

Comment:
This feels kind of useless for projects that are essentially shared through npm. I'd suggest removing it altogether from the TL;DR and burying it somehwere deep, noting that they're not really relevant to JS.

Reply:
> Thanks, @brianwarner.
> 
> I think we still have some fundamental disagreements about what maintainer-friendly implies. ;) :D
> 
> We should be aiming to build a trust-based relationship between the maintainers and the foundation, where the maintainers rely on your expertise to interpret the IP policy, distilled in that document, and have simple guidelines to follow.
> 
> I feel like the added disclaimers and explanations seem designed to protect the foundation rather than serve the projects, and end up feeling adversarial to maintainers rather than helpful.
> 
> I'll go through the doc and offer a pared-down version that's really focused on helping maintainers onboard their projects and we'll see if we can meet somewhere in the middle (ETA end of next week earliest).
> 
> How does that sound?

That sounds great, thanks @tobie.  The pen is yours!

My only request is that we keep enough disclaimers and explanations to avoid introducing confusion about which document governs.  I'd like to minimize the risk of a contributor looking at this and thinking, "Ah, so that's the entire IP policy," and end up missing something important.

That said, I like the way you position it, and am absolutely here and happy to help maintainers find answers to their questions on the upstream policy.

Thanks for your involvement, looking forward to seeing the next iteration.